MapboxNavigation/NavigationMapView.swift

```diff
@@ -519,6 +521,16 @@ open class NavigationMapView: MGLMapView {
             style.removeLayer(arrowCasingSymbolLayer)
         }
     }
+    
+    func mutateBaseCameraStops(multiplier: Double) -> [Int: MGLStyleValue<NSNumber>] {
```

Comment:
I think this method would be more generally useful as an extension on `MGLStyleValue<NSNumber>` or `[Int: MGLStyleValue<NSNumber>]`. Something like this:

```swift
// Extends a dictionary of camera function stops
extension Dictionary<Key, Value> where Key: Numeric, Value: MGLStyleValue<NSNumber> {
  func muliplied(by factor: Double) -> Self {
    …
  }
}
```

in order to say something like this:

```swift
routeLineWidthAtZoomLevels.multiplied(by: 0.9)
```